libtcmu.c

```diff
@@ -277,78 +268,43 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 	case MODE_SELECT:
 	case MODE_SELECT_10:
 		return tcmu_emulate_mode_select(cdb, iovec, iov_cnt, sense);
+	default:
+		tcmu_err("unknown command %x\n", cdb[0]);
+		return TCMU_NOT_HANDLED;
+	}
+}
+
+static bool command_is_store_call(uint8_t cmd)
+{
```

Comment:
How do we decide which is store call cmd? And it is overkill to choose one as store call cmd. for e.g. In this patch you make compare and write as store call cmd then all the handlers must implement compare_and_write cmd. But with passthrough mechanism, only the handler that pass down the cmd should handle it. If any one of the handler decide the cmd is a store cmd, then all the others would handle it, this break the reason we abstract out SCSI details from store drivers.